src/main/java/cinemas/controllers/user/VouchersController.java

```diff
@@ -0,0 +1,16 @@
+package cinemas.controllers.user;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
+@RequestMapping("/users/vouchers")
+public class VouchersController {
+    @GetMapping("/user_id")
```

Comment:
```suggestion
    @GetMapping("/user_id")
```
chỗ này sai nha, user_id là path variable đó bé 
ex url thật: users/1/vouchers -> all voucher của user có id là 1
users/3/vouchers -> all voucher của user có id là 3